### ISSUE TITLE
bug: CLI Init Commands don't Work OOTB in PowerShell

### ISSUE THREAD

**Issue by erin-allison** [triggerdotdev/trigger.dev#881]
### Provide environment information

System:
- OS: Windows 11 10.0.22621
- CPU: (16) x64 AMD Ryzen 7 7800X3D 8-Core Processor
- Memory: 27.07 GB / 63.15 GB

Binaries:
- Node: 18.19.0 - C:\Program Files\nodejs\node.EXE
- npm: 10.2.3 - C:\Program Files\nodejs\npm.CMD
- pnpm: 8.15.1 - ~\AppData\Roaming\npm\pnpm.CMD

### Describe the bug

In PowerShell, a token beginning with the '@' character is used for [splatting](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_splatting?view=powershell-7.4), and is invalid for use when calling a standard executable.

For a (trivially) better DX, the package name in that command should be enclosed in quotes so that the command doesn't need to be edited before execution in PowerShell.

![image](https://github.com/triggerdotdev/trigger.dev/assets/75811601/e4afcdb4-4581-4cf8-9d8e-d468564638e7)

![image](https://github.com/triggerdotdev/trigger.dev/assets/75811601/ca8ed424-e3f7-4888-9656-6cd014840b7c)

### Reproduction repo

n/a

### To reproduce

1. Go to Framework Setup and select a framework
2. Copy and paste the "Run the CLI 'init' command..." step into a PowerShell window

**Comment by matt-aitken** [comment]
This is a good note 👍 

For v3 the CLI will be called just `trigger.dev`. Will that work ok with Powershell @erin-allison? I assume dots are ok? Otherwise we could call it `triggerdotdev`.

**Comment by erin-allison** [comment]
Yes, dots are okay, and "@" signs in the middle of a string are fine, just not as the first character of a token.

**Comment by matt-aitken** [comment]
Thanks! We didn't realise this until our first Powershell user started the CLI. It explains why CLIs are never namespaced packages on npm.

**Comment by matt-aitken** [comment]
Thanks! We didn't realise this until our first Powershell user started the CLI. It explains why CLIs are never namespaced packages on npm.

**Comment by matt-aitken** [comment]
This isn't a problem in v3+, we renamed the CLI to `trigger.dev`. v2 is deprecated as of Jan 31 2025
